src/sail_lean_backend/Sail/Sail.lean

```diff
@@ -15,5 +15,16 @@ def truncate {w: Nat} (x: BitVec w) (w': Nat) : BitVec w' :=
 def truncateLSB {w: Nat} (x: BitVec w) (w': Nat) : BitVec w' :=
   x.extractLsb' 0 w'
```

Comment:
```suggestion
def truncateLsb {w: Nat} (x: BitVec w) (w': Nat) : BitVec w' :=
  x.extractLsb' 0 w'
```

We generally treat `Lsb` as one word, for the sake of capitalization. (Outside of the diff of this PR, so feel free to ignore)